plugins/woocommerce-admin/client/marketplace/components/category-selector/category-selector.tsx

```diff
@@ -131,50 +215,60 @@ export default function CategorySelector(
 
 	return (
 		<>
-			<ul className="woocommerce-marketplace__category-selector">
-				{ visibleItems.map( ( category ) => (
+			<ul
+				className="woocommerce-marketplace__category-selector"
+				aria-label="Categories"
+				ref={ categorySelectorRef }
+			>
+				{ categoriesToShow.map( ( category ) => (
 					<li
 						className="woocommerce-marketplace__category-item"
 						key={ category.slug }
+						ref={
+							category.slug === selected?.slug
+								? selectedCategoryRef
+								: null
+						}
 					>
 						<CategoryLink
 							{ ...category }
 							selected={ category.slug === selected?.slug }
+							aria-current={ category.slug === selected?.slug }
 						/>
 					</li>
 				) ) }
-				<li className="woocommerce-marketplace__category-item">
-					{ dropdownItems.length > 0 && (
-						<CategoryDropdown
-							type={ props.type }
-							label={ __( 'More', 'woocommerce' ) }
-							categories={ dropdownItems }
-							buttonClassName={ clsx(
-								'woocommerce-marketplace__category-item-button',
-								{
-									'woocommerce-marketplace__category-item-button--selected':
-										isSelectedInDropdown(),
-								}
-							) }
-							contentClassName="woocommerce-marketplace__category-item-content"
-							arrowIconSize={ 20 }
-							selected={ selected }
-						/>
-					) }
-				</li>
 			</ul>
-
 			<div className="woocommerce-marketplace__category-selector--full-width">
 				<CategoryDropdown
 					type={ props.type }
 					label={ mobileCategoryDropdownLabel() }
-					categories={ visibleItems.concat( dropdownItems ) }
+					categories={ categoriesToShow }
 					buttonClassName="woocommerce-marketplace__category-dropdown-button"
 					className="woocommerce-marketplace__category-dropdown"
 					contentClassName="woocommerce-marketplace__category-dropdown-content"
 					selected={ selected }
 				/>
 			</div>
+			{ isOverflowing && (
+				<>
+					<button
+						onClick={ scrollToPrevCategories }
+						className="woocommerce-marketplace__category-navigation-button woocommerce-marketplace__category-navigation-button--prev"
+						hidden={ scrollPosition === 'start' }
+						aria-label="Scroll to previous categories"
+					>
+						<Icon icon="arrow-left-alt2" />
+					</button>
```

Comment:
These buttons are overlayed on top of the category list to 

1) indicate there are more categories to see
2) help people scroll

In terms of accessibility and tab navigation, it's a bit clunky. I thought about removing them from the tab index but didn't act n it to see hear what you think

Reply:
See my main review comment...I agree they should be removed from the tab index. Keyboard nav through the categories work fine without using the buttons at all.

Reply:
Awesome, thank you for checking this! I have implemented it using `tabIndex={ -1 }`